Dockerfile

```diff
@@ -31,7 +31,10 @@ RUN echo 'deb http://www.rabbitmq.com/debian testing main' > /etc/apt/sources.li
 
 ENV RABBITMQ_VERSION 3.5.6-1
 
-RUN apt-get update && apt-get install -y rabbitmq-server=$RABBITMQ_VERSION --no-install-recommends && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && \
+	apt-get install -y erlang erlang-mnesia erlang-public-key erlang-crypto erlang-ssl \
+		erlang-asn1 erlang-inets erlang-os-mon erlang-xmerl erlang-eldap && \
+	apt-get install -y rabbitmq-server=$RABBITMQ_VERSION --no-install-recommends && rm -rf /var/lib/apt/lists/*
```

Comment:
This is formatted kind of strangely (any particular reason for two `apt-get install` invocations?) -- can we re-flow to something like this, to match the rest of the file:
```dockerfile
# explicit erlang package installation for TLS support
RUN apt-get update && apt-get install -y --no-install-recommends \
		erlang erlang-mnesia erlang-public-key erlang-crypto erlang-ssl erlang-asn1 erlang-inets erlang-os-mon erlang-xmerl erlang-eldap \
		rabbitmq-server=$RABBITMQ_VERSION \
	&& rm -rf /var/lib/apt/lists/*
```